sopel/plugins/handlers.py

```diff
@@ -128,47 +130,47 @@ def is_loaded(self):
         raise NotImplementedError
 
     def setup(self, bot):
-        """Setup the plugin with the ``bot``
+        """Run the plugin's setup routine.
 
         :param bot: instance of Sopel
         :type bot: :class:`sopel.bot.Sopel`
         """
         raise NotImplementedError
 
     def has_setup(self):
-        """Tell if the plugin has a setup action
+        """Tell if the plugin has a setup routine.
 
         :return: ``True`` if the plugin has a setup, ``False`` otherwise
         :rtype: bool
         """
         raise NotImplementedError
 
     def register(self, bot):
-        """Register the plugin with the ``bot``
+        """Register the plugin with the ``bot``.
 
         :param bot: instance of Sopel
         :type bot: :class:`sopel.bot.Sopel`
         """
         raise NotImplementedError
 
     def unregister(self, bot):
-        """Unregister the plugin from the ``bot``
+        """Unregister the plugin from the ``bot``.
 
         :param bot: instance of Sopel
         :type bot: :class:`sopel.bot.Sopel`
         """
         raise NotImplementedError
 
     def shutdown(self, bot):
-        """Take action on bot's shutdown
+        """Run the plugin's shutdown routine.
 
         :param bot: instance of Sopel
         :type bot: :class:`sopel.bot.Sopel`
         """
         raise NotImplementedError
 
     def has_shutdown(self):
-        """Tell if the plugin has a shutdown action
+        """Tell if the plugin has a shutdown routine.
```

Comment:
Ditto my comment explaining the suggestion on `setup()`.

```suggestion
        """Tell if the plugin has a shutdown action.
```